apps/api/src/modules/catalog/infrastructure/queries/watching-shows-count.query.spec.ts

```
@@ -0,0 +1,55 @@
+import { WatchingShowsCountQuery } from './watching-shows-count.query';
+import { DatabaseException } from '../../../../common/exceptions/database.exception';
+
+const createThenable = (resolveWith: any = [], rejectWith?: Error) => {
+  const thenable: any = {};
+  const methods = ['select', 'from', 'innerJoin', 'where'];
+  methods.forEach((m) => {
+    thenable[m] = jest.fn().mockReturnValue(thenable);
+  });
+
+  if (rejectWith) {
+    thenable.then = (_res: any, rej: any) => Promise.reject(rejectWith).catch(rej);
```

Biome (2.4.4)

[error] Do not add then to an object.

(lint/suspicious/noThenProperty)